src/api/apiService.js

```diff
@@ -2,53 +2,73 @@ import { sessionService } from 'redux-react-native-session';
 import humps from 'humps';
 import Config from 'react-native-config';
 
-const saveSessionHeaders = (headers) => {
-  if (headers.get('access-token')) {
-    const sessionHeaders = {
-      token: headers.get('access-token'),
-      uid: headers.get('uid'),
-      client: headers.get('client')
-    };
-    sessionService.saveSession(sessionHeaders);
-  }
+import handleErrors from './utils/handleErrors';
+import getResponseBody from './utils/getResponseBody';
```

Comment:
Can you put absolute imports here?

Reply:
I think it doesn't make sense here since it's on the same folder, wdyt? but I like the idea of using absolute imports 👍